<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widgetCalendar</class>
 <widget class="QWidget" name="widgetCalendar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>校历</string>
  </property>
  <widget class="QLabel" name="labSemester">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>学期</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editSemester">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labStartDate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>学期开始</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateStart">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEnd">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labEndDate">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>学期结束</string>
   </property>
  </widget>
  <widget class="QLabel" name="labHoliday">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>假期安排</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="editHoliday">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>新增校历条目</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnModify">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>修改选中条目</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>删除选中条目</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableCalendar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>531</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>保存校历数据</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>展示所有校历条目</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
